src/libraries/Microsoft.PowerFx.Core/FeatureFlags.cs

```diff
@@ -25,13 +31,28 @@ public static bool StringInterpolation
             set
             {
                 // Once flipped, can't unflip. 
-                if (!value) 
+                if (!value && !_inTests) 
                 { 
                     throw new NotSupportedException("Can't change field after it's set");
                 }
 
                 _stringInterpolation = value;
             }
         }
+
+        public static bool NoValueInRecordArrays
```

Comment:
naming: `TableSyntaxDoesntWrapRecords`? 
Basically, we don't have arrays, and we shouldn't mention "array" on our public surface.

Reply:
Ok. We'll soon have to support Arrays I think

Reply:
Discussed in Greg's recent xr: there is no plan to support arrays in PowerFx at the moment. If this change isn't sufficient, we may also support default fields within records, but beyond that, an array type is unlikely.